### PR TITLE
Leaderboard app with query

### LINKED ISSUE #1630: Users SBAT to see TGSTO leaderboard in the web app
Implement the leaderboard according to https://www.figma.com/file/HzDfZq4jvFNgUvPuPmeC80Ss/Website-3.0?node-id=1919%3A0 in the celo web page

### PR DESCRIPTION
**Pull request by diminator**
### Description

This PR requires branch `aaronmgdr/build4`

Implements the dynamic leaderboard app using a graphql query and leaderboard script:
https://github.com/celo-org/celo-monorepo/pull/1810

### Tested

Tested by running locally.

### Related issues

- Fixes #1630 

### To check

I'm not sure how `env-config.js.template` works

### CODE REVIEW

File: packages/web/src/dev/LeaderBoard.tsx [hunk 1]
```diff
@@ -97,10 +94,11 @@ function sorter(alpha: Competitor, bravo: Competitor) {
   return -1
 }
 
-const PORTIONS = 8
+const Portions = 8
```
Comment (aaronmgdr) [inline packages/web/src/dev/LeaderBoard.tsx:97]: why the change? for consts all caps is really more common, with Pascalcase I tend to think interfaces or classes

File: packages/web/env-config.js.template [hunk 2]
```diff
@@ -9,7 +9,7 @@ const environments = {
       projectId: 'celo-faucet',
     },
     LEADERBOARD: {
-      uri: "http://localhost:8080/v1/graphql",
+      uri: "http://localhost:4000/graphiql",
```
Comment (aaronmgdr) [inline packages/web/env-config.js.template:12]: I assume eventually this will be a non localhost URL?

**Review by aaronmgdr — APPROVED** [review]

File: packages/web/env-config.js.template (hunk 2 above)
Comment (diminator) [reply packages/web/env-config.js.template:12]: yes, it should be the one of `blockscout`

File: packages/web/src/dev/LeaderBoard.tsx (hunk 1 above)
Comment (diminator) [reply packages/web/src/dev/LeaderBoard.tsx:97]: true, the linter was complaining, but we can revert

**Comment by diminator** [comment]
Seems circleci is having some issues. probably `build4` branch needs to sync in master
